### PR TITLE
Fix moveend firing when zooming out/in at min/max

### PR DESCRIPTION
**Pull request by ben-lu-uw**
Closes issue https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/550

Redefines onKeyDown to have this line
https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/5c649039b6cace1ee0303f4540a6c9ae49eca2cb/src/mapml/keyboard.js#L38

### CODE REVIEW

File: src/mapml/keyboard.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        } else if (key === 27 && map._popup && map._popup.options.closeOnEscapeKey) {
+            map.closePopup();
+
+        } else {
+            return;
+        }
+
+        stop(e);
+    }
+});
```
Comment (prushforth) [inline src/mapml/keyboard.js:50]: I have seen this a few times, always wondered what the significance was, so I [googled it.](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)  I guess we should try to add newlines at end of file where appropriate, i.e. a blank line at the end of the file

**Review by prushforth — COMMENTED** [review]
Question: this keyboard include looks interesting - did you find an example of this in Leaflet?  Could we use this technique to provide modifiers for the arrow keys when they are Shift-arrow or Ctrl-Shift-arrow for instance?  I have always thought it would be good to provide the ability to reduce or increase the pan amount by providing such modifier key mappings.  Anyway looks good, code is well organized to my eye.  I tried it and could not reproduce the problem, but let's review together and you can show me where I'm missing something.

**Comment by ben-lu-uw** [comment]
We can change the modifier for the pan here except my redefine seems to have broken it since toPoint is not defined :
https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/5c649039b6cace1ee0303f4540a6c9ae49eca2cb/src/mapml/keyboard.js#L24-L31

**Comment by prushforth** [comment]
Sorry, I meant can **this** be merged?

**Comment by ahmadayubi** [comment]
> Question: this keyboard include looks interesting - did you find an example of this in Leaflet? Could we use this technique to provide modifiers for the arrow keys when they are Shift-arrow or Ctrl-Shift-arrow for instance?

Some leaflet documentation that may come in use for this functionality are:

Panning Delta - https://leafletjs.com/reference.html#map-keyboardpandelta

Zooming Delta - https://leafletjs.com/reference.html#map-zoomdelta

File: src/mapml/keyboard.js (hunk 1 above)
Comment (prushforth) [reply src/mapml/keyboard.js:50]: LGTM! Thanks @ben-lu-uw

**Review by prushforth — APPROVED** [review]

**Pull request merged**